recipes/utide/meta.yaml

```diff
@@ -0,0 +1,39 @@
+{% set version = "0.1.0" %}
+{% set commit = "0c612d557f615fbbb7d513892b6840b2302b65e7" %}
+
+package:
+    name: utide
+    version: {{ version }}.dev0
+
+source:
+    fn: {{ commit }}.tar.gz
+    url: https://github.com/wesleybowman/UTide/archive/{{ commit }}.tar.gz
+    sha256: 1000aa9c552309e8e4f5a20f9ed2e62cd1c6bc3794476f5aacc25c2326d6fe08
+
+build:
+    number: 0
+    skip: True  # [win and np>=111]
```

Comment:
We would get an unresolved dependencies. Not sure how you would fix that in conda-build-all... Sounds like a difficult task.